### PR TITLE
add family to cldfbench script

### PR DESCRIPTION
**Pull request by FredericBlum**
@xrotwang I tried adding LanguageFamily to the cldf/languages.csv, but for some reason, it does not get added. The print statement I added to the cldfbench-script while iterating through `raw/languages.csv` works fine, but the LanguageTable does not receive a column `Family`. Could you have a look what I am doing wrong?

If you agree, I'd also add myself to the list of contributors to the dataset.

### CODE REVIEW

**Comment by xrotwang** [comment]
Will do.
From a cursory glance, I'd say the problem is that "Family" is not added to the dataset schema - and only columns listed in the metadata are written to file. So you'd need to add a "Family" column here: https://github.com/cldf-datasets/doreco/blob/8a9e4b604e9b4840556dc94dd224693c53f7a6f7/cldfbench_doreco.py#L484-L489

**Comment by xrotwang** [comment]
Btw.: If we create a 1.2.1 release for this dataset, we'll have to make sure to limit the actual data to the corpora with appropriate licenses, as we did for 1.2.

**Comment by FredericBlum** [comment]
Thanks, I have managed to include Family now.

The last commit also removes the nd-languages.

**Comment by FredericBlum** [comment]
Ah, of course, I re-downloaded, but not delete the files previously so they all stayed there... thanks for checking on this! Are we good to merge now?

**Comment by xrotwang** [comment]
Yes. I think so.

Frederic Blum ***@***.***> schrieb am Mi., 17. Apr. 2024,
22:08:

**Comment by xrotwang** [comment]
@FredericBlum I'll add a CHANGES.md and do a release 1.2.1, ok?

**Comment by FredericBlum** [comment]
Great, thanks!

File: cldfbench_doreco.py [hunk 1]
```diff
@@ -220,12 +220,10 @@ def cmd_makecldf(self, args):
                 "dc:bibliographicCitation": CORPUS_CITATION_FMT.format(**row)
             })
 
-            print(row["Language"], row["Family"])
             args.writer.objects["LanguageTable"].append({
                 "ID": row["Glottocode"],
                 "Name": row["Language"],
                 "Glottocode": row["Glottocode"],
-                "Family": row["Family"],
```
Comment (xrotwang) [inline cldfbench_doreco.py]: We got a funny regression here. Now we add "Family" to the schema - but not to the data :) Will fix.

**Pull request merged**